build.py

```diff
@@ -755,7 +551,14 @@ def create_dockerfile_build(ddir, dockerfile_name, argmap):
 FROM tritonserver_builder_image AS build
 FROM tritonserver_buildbase
 COPY --from=build /tmp/tritonbuild /tmp/tritonbuild
+
+# Copy ONNX custom op library and model (Needed for testing)
```

Comment:
Should be inside the ` if 'onnxruntime' in backends:` conditional. Causes Windows CI to fail